README.md

```diff
@@ -33,17 +33,7 @@ In general, this project follows [Terraform Versioning Specification](https://ww
 
 ## Contributors
 
-Pull requests, issues and comments are welcomed. For pull requests:
-
-* Add tests for new features and bug fixes
-* Follow the existing style
-* Separate unrelated changes into multiple pull requests
-
-See the existing issues for things to start contributing.
-
-For bigger changes, make sure you start a discussion first by creating an issue and explaining the intended change.
-
-JFrog requires contributors to sign a Contributor License Agreement, known as a CLA. This serves as a record stating that the contributor is entitled to contribute the code/documentation/translation to the project and is willing to have it used in distributions and derivative works (or is willing to transfer ownership).
+See the [contribution guide](https://github.com/jfrog/terraform-provider-pipeline/blob/master/CONTRIBUTIONS.md).
```

Comment:
I think you can use relative path here so avoid future path issue, or when users fork the repo.